<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_AddSupplier</class>
 <widget class="QFrame" name="F_AddSupplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutMain">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutLeftSide">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="L_company">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Company *</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_company">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>90</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_name">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="labelText">
          <string>Name *</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_vtaNumber">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="labelText">
          <string>VTA Number</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_email">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="labelText">
          <string>Email Address *</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_phone">
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="labelText">
          <string>Phone *</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="L_info">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="layoutRightSide">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="L_country">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Country</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_country">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="maxCount">
          <number>300</number>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_city">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>524287</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="labelText">
          <string>City *</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_state">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>524287</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="labelText">
          <string>State</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_address">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>524287</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="labelText">
          <string>Address *</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MDLineEdit" name="W_postalcode">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>75</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>524287</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="labelText">
          <string>Postal Code</string>
         </property>
         <property name="helperVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="leftIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="rightIconVisible" stdset="0">
          <bool>false</bool>
         </property>
         <property name="fontSize" stdset="0">
          <number>15</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_addSupplier">
         <property name="text">
          <string>Add Supplier</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MDLineEdit</class>
   <extends>QWidget</extends>
   <header>mdlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
